### PR TITLE
[WIP] Early Access PGE

### PR DESCRIPTION
**Pull request by rachelwhitton**
## Effect
PR includes the following changes:
- Early Access warning on targeted page
- First pass on Early Access doc

## Remaining Work
- [ ] Technical Review from @ari-gold 
- [ ] Copy review from @alexfornuto or @allenfear

### CODE REVIEW

File: source/_docs/free-https.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+      <sup><a href="http://caniuse.com/#search=TLS%201.2">1 <a href="http://caniuse.com/#search=SNI">2</a></sup></td>
+    </tr>
+    <tr>
+      <th><a href="https://www.ssllabs.com/ssltest/">SSL Labs</a> Rating</th>
+      <td>A</td>
+      <td>A+ <a href="#http-strict-transport-security-header">with HSTS</a></td>
+    </tr>
+    <tr>
+      <th>Protocol</th>
+      <td>Vulnerable TLS 1.1</td>
```
Comment (garydylina) [inline source/_docs/free-https.md]: This should be "TLS 1.1 & 1.2" vs. "Vulnerable TLS 1.1" unless someone is aware of an inherent TLS 1.1 vulnerability.

File: source/_docs/free-https.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+      <td>TLS 1.2 only</td>
+    </tr>
+    <tr>
+      <th>Ciphers</th>
+      <td>Vulnerable 3DES cipher</td>
```
Comment (garydylina) [inline source/_docs/free-https.md]: I prefer that we use the term "Weak 3DES cipher" vs "Vulnerable 3DES cipher"

**Comment by hengkit** [comment]
2 things stand out to me but aren't critical. 
1) Maybe start with an optional "Set your DNS TTL to a low number"
2) 25 - 60 minutes seems arbitrary, maybe just say "up to an hour"

and there's an `HTTTPS` somewhere in the middle

File: source/_docs/free-https.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
+---
+title: Early Access: Free and Automated HTTPS
+earlyaccess: true
+description: Upgrade to Free and Automated HTTPS, powered by Let's Encrypt
+---
+Upgrade your sites's HTTPS and never manage certificates again. Pantheon automatically adds all of your site's domains to a shared certificate and  serves it through our globally distributed content delivery network (CDN). Upgrade to take advantage of performance, security benefits, plus the time and cost of manually purchasing and maintaining certs.
```
Comment (kibra) [inline source/_docs/free-https.md]: site's -> sites' 2x

File: source/_docs/free-https.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+### Does upgrading involve HTTPS interruptions or downtime?
+No, after you update your DNS records, traffic will gracefully switch over and involves no downtime or HTTPS interruption.
+
+**Caveat:** If after upgrading you add a new domain that is not already routed to Pantheon, then it will take 25 minutes to an hour for HTTPS to be ready for that new domain. Pre-provisioning HTTPS for new domains is planned after early access, with the full release.
+
+### Which browsers and operating systems are supported?
+All modern browsers and operating systems are supported. For details, see the **Handshake Simulation** portion of this [report](https://www.ssllabs.com/ssltest/analyze.html?d=pantheon.io).
+
+### When do I stopped being billed $30/month?
+Pantheon will remove legacy load balancers and stop billing within 30 days of upgrading.
```
Comment (kibra) [inline source/_docs/free-https.md]: Can we clarify this a little?

Once we stop charging (i.e. the LBs are deleted) we won't be able to automatically revert the customer. Do we want to say "after 30 days of upgrading"? Or "within 60 days, with the ability to revert for 30 days" ?

File: source/_docs/free-https.md (hunk 3 above)
Comment (kibra) [inline source/_docs/free-https.md]: Can we make it more clear "globally distributed content delivery network" is a new thing?

File: source/_docs/free-https.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
+---
+title: Early Access: Free and Automated HTTPS
+earlyaccess: true
+description: Upgrade to Free and Automated HTTPS, powered by Let's Encrypt
+---
+Upgrade your sites's HTTPS and never manage certificates again. Pantheon automatically adds all of your site's domains to a shared certificate and  serves it through our globally distributed content delivery network (CDN). Upgrade to take advantage of performance, security benefits, plus the time and cost of manually purchasing and maintaining certs.
+
+## Eligibility
+Invite only. As of April 2017, sites on a Professional plan with HTTPS already enabled and all custom domains routing to Pantheon may [request an invite](http://learn.pantheon.io/201701-HTTPS-Reg.html).
```
Comment (kibra) [inline source/_docs/free-https.md]: Invites are per-site, not per user. Not sure if it's worth clearifying this.

File: source/_docs/free-https.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+3. Once HTTPS is ready, you will see action required on the **Domains & HTTPS** tab for each environment, with instructions to configure DNS for each custom domain.  
+4. Click **Show DNS Recommendations** next to each custom domain to identify DNS values needed to point the domain to your site. Domains that are not yet configured will indicate action is required.
+5. Configure DNS using the provided destinations at the domain's DNS provider.
+
+    <div class="alert alert-info">
+    <h3 class="info">Pro Tip</h3>
+Look up your DNS provider with this free web tool: <a href="https://mxtoolbox.com/DNSLookup.aspx">https://mxtoolbox.com/DNSLookup.aspx</a>
+</div>
+
+6. Wait for your DNS changes to fully propagate. DNS records are cached across the internet and can take up to 72 hours to propagate, depending on the time to live (TTL) configured for the domain's DNS records.
```
Comment (kibra) [inline source/_docs/free-https.md]: Where did we get 72 hours from?

File: source/_docs/free-https.md [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+      <td>Bring your own</td>
+      <td>Shared, issued by Let's Encrypt</td>
+    </tr>
+    <tr>
+      <th>Renewal</th>
+      <td>Manual</td>
+      <td>Automatic</td>
+    </tr>
+    <tr>
+      <th>Unique inbound IP Address</th>
```
Comment (kibra) [inline source/_docs/free-https.md]: They will still have an A record to point to on "Free & Automated"
Maybe something like "Inbound IP: Unique   |  Shared"

Comment (ari-gold) [inline source/_docs/free-https.md]: +1 @rachelwhitton Can you please update?

File: source/_docs/free-https.md (hunk 4 above)
Comment (ari-gold) [inline source/_docs/free-https.md]: https://github.com/pantheon-systems/documentation/pull/2323

File: source/_docs/free-https.md (hunk 6 above)
Comment (ari-gold) [inline source/_docs/free-https.md]: the internets, e.g. https://support.google.com/a/answer/47283?hl=en

Comment (kibra) [inline source/_docs/free-https.md]: Cool. I like the wording here a little better: "can take up to 72 hours to go into effect, but typically they happen much sooner."

I just don't want anything to think they *need* to wait, especially if the tool below shows global propagation.

File: source/_docs/free-https.md [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+High grade TLS 1.2 encryption with up-to-date ciphers. For a deep analysis of the HTTPS configuration on upgraded sites see [this A+ SSL Labs report for https://pantheon.io](https://www.ssllabs.com/ssltest/analyze.html?d=pantheon.io).
+
+### How can I obtain an A+ SSL Labs rating?
+Follow the steps to upgrade and send the [HSTS header](#http-strict-transport-security-header) as described above.
+
+### Are wildcard certificates supported?
+No, but you don’t need a wildcard certificate to secure communications for multiple domains because we will automatically deploy certificates for all domains on your site.
+
+### Is Extended Validation supported?
+No, please take a moment to fill out the [HTTPS survey](/docs/getting-support) if you require Extended Validation.
```
Comment (ari-gold) [inline source/_docs/free-https.md]: https://www.getfeedback.com/r/LETtb3QV

**Comment by ari-gold** [comment]
+1

**Pull request merged**